globus_automate_client/cli/flows.py

```diff
@@ -458,16 +447,29 @@ def flow_display(
     verbose: bool = verbosity_option,
 ):
     """
-    Display a deployed Flow. You must have either created the Flow or
-    be present in the Flow's "visible_to" list to view it.
+    Visualize a local or deployed Flow defintion. If providing a Flows's ID, You
+    must have either created the Flow or be present in the Flow's "visible_to"
+    list to view it.
     """
-    fc = create_flows_client(CLIENT_ID, flows_endpoint)
-    try:
-        flow_get = fc.get_flow(flow_id)
-    except GlobusAPIError as err:
-        format_and_echo(err, verbose=verbose)
+    if not flow_definition and not flow_id:
+        raise typer.BadParameter("Either FLOW_ID or --flow_definition should be set.")
+    if flow_definition and flow_id:
+        raise typer.BadParameter(
+            "Only one of FLOW_ID or --flow_definition should be set."
+        )
```

Comment:
nice, always good to provide more validation and user messages about what went wrong